### PR TITLE
Named map widgets fixes

### LINKED ISSUE #6681: vizJSON (v3) inconsistent on a page reload
STR:
1. enter the new editor with a vis that has a torque layer
1. add any widget
1. reload

Expected result:
- the time-series widget working

Actual result:
- not only the time-series fails, but all widgets, because the underlying request to get the dataviews data fails with a 400 error `{errors: ["cartocss can't be undefined"]}`
- the layer data provided in the inlined vizJSON (end of the full-page response) doe't contained the expected `cartocss` or `sql` props on the torque layer (they're null instead).
- reloading 2nd time and things are correct again

### PR DESCRIPTION
**Pull request by juanignaciosl**
This fixes #6681 and #6691.

### CODE REVIEW

**Comment by juanignaciosl** [comment]
@gfiorav , CR this, please.

FYI, @rochoa , @javisantana , I've had to make a couple of transformations, please take a look at `named_map.rb` changes (`aggregationColumn`/`aggregation_column` and widget type).

cc @viddo , @xavijam 

PS: @xavijam this is currently deployed in ded02. Could you confirm that you can't reproduce #6691 any more? I managed to break named with some interactions, but this PR should fix them as well.

File: app/controllers/carto/api/vizjson3_presenter.rb [hunk 1]
```diff
@@ -29,6 +28,12 @@ def to_vizjson(vector: false, **options)
 
       private
 
+      def symbolize_vizjson(vizjson)
+        vizjson = vizjson.deep_symbolize_keys
+        vizjson[:layers] = vizjson[:layers].map(&:deep_symbolize_keys)
+        vizjson
+      end
```
Comment (gfiorav) [inline app/controllers/carto/api/vizjson3_presenter.rb]: how come you can't add the layers and then `.deep_symbolize_keys` everything at the end?

Comment (juanignaciosl) [inline app/controllers/carto/api/vizjson3_presenter.rb]: This is not read that way. I'm not _adding_ layers, but _replacing `:layers` content with "deep-symbolized-:layers". This is a workaround for a `deep_symbolize_keys` limitation: it doesn't symbolize array contents. `vizjson[:layers]` is an array with hashes having string keys which are not symbolized unless I do it this way.

Comment (gfiorav) [inline app/controllers/carto/api/vizjson3_presenter.rb]: ah, I see! Ok :sparkles:

**Comment by gfiorav** [comment]
Kooks good to me :+1:

**Comment by gfiorav** [comment]
oops!

**Comment by xavijam** [comment]
After testing it it looks good :+1:.

**Comment by juanignaciosl** [comment]
@gfiorav CR this again, please. I had to add a refactor to make widget information loading testable again.

**Comment by gfiorav** [comment]
:+1:

**Pull request merged**